osu.Game/Database/LegacyBeatmapImporter.cs

```diff
@@ -34,9 +33,9 @@ protected override IEnumerable<string> GetStableImportPaths(Storage storage)
 
                     try
                     {
-                        if (!directoryStorage.GetFiles(string.Empty).ExcludeSystemFileNames().Any())
+                        if (!directoryStorage.GetFiles(string.Empty, "*.osu").Any())
```

Comment:
I guess the reason for removal of `.ExcludeSystemFileNames()` is that the wildcard pattern excludes them already...?